### PR TITLE
Rubocop cleanup

### PR DESCRIPTION
**Pull request by james2m**
Adds a Rubocop config for 0.48.1 based on the config from grape,  and update the todo list for the existing offences. I don't have time to fix all those issues so I suggest tackling the todo should be a separate PR.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,6 +1,30 @@
 AllCops:
+  TargetRubyVersion: 2.1
```
Comment (LeFnord) [inline .rubocop.yml]: please change it to 2.4, maybe alos in the grape

**Comment by LeFnord** [comment]
please run `bundle exec rubocop -a` to fix the frozen string luteral stuff, thanks

**Comment by LeFnord** [comment]
thanks @james2m

**Comment by dblock** [comment]
FWIW I generally just stick to `rubocop -a ; rubocop --auto-gen-config`. It's easier to think about.

**Pull request merged**